internal/testing/testing.go

```diff
@@ -0,0 +1,765 @@
+}
+
+func RequireEqualf(t *testing.T, expected interface{}, actual interface{}, msg string, args ...interface{}) {
+	t.Helper()
+
+	if Equalf(t, expected, actual, msg, args...) {
+		return
+	}
+
+	t.FailNow()
+}
+
+func Contains(t *testing.T, s, contains string, msgAndArgs ...interface{}) bool {
+	t.Helper()
+
+	if !strings.Contains(s, contains) {
+		return fail(t, fmt.Sprintf("%#v does not contain %#v", s, contains), msgAndArgs...)
+	}
+
+	return true
+}
+
+func Containsf(t *testing.T, s, contains string, msg string, args ...interface{}) bool {
+	t.Helper()
+
+	return Contains(t, s, contains, append([]interface{}{msg}, args...)...)
+}
+
+func RequireContains(t *testing.T, s, contains string, msgAndArgs ...interface{}) {
+	t.Helper()
+
+	if Contains(t, s, contains, msgAndArgs...) {
+		return
+	}
+
+	t.FailNow()
+}
+
+func RequireContainsf(t *testing.T, s, contains string, msg string, args ...interface{}) {
+	t.Helper()
+
+	if Containsf(t, s, contains, msg, args...) {
+		return
+	}
+
+	t.FailNow()
+}
+
+func NotContains(t *testing.T, s, contains string, msgAndArgs ...interface{}) bool {
+	t.Helper()
+
+	if strings.Contains(s, contains) {
+		return fail(t, fmt.Sprintf("%#v should not contain %#v", s, contains), msgAndArgs...)
+	}
+
+	return true
+}
+
+func RequireNotContains(t *testing.T, s, contains string, msgAndArgs ...interface{}) {
+	t.Helper()
+
+	if NotContains(t, s, contains, msgAndArgs...) {
+		return
+	}
+
+	t.FailNow()
+}
+
+func True(t *testing.T, value bool, msgAndArgs ...interface{}) bool {
+	if !value {
+		t.Helper()
+		return fail(t, "Should be true", msgAndArgs...)
+	}
+
+	return true
+}
+
+func Truef(t *testing.T, value bool, msg string, args ...interface{}) bool {
+	t.Helper()
+
+	return True(t, value, append([]interface{}{msg}, args...)...)
+}
+
+func RequireTrue(t *testing.T, value bool, msgAndArgs ...interface{}) {
+	t.Helper()
+
+	if True(t, value, msgAndArgs...) {
+		return
+	}
+
+	t.FailNow()
+}
+
+func RequireTruef(t *testing.T, value bool, msg string, args ...interface{}) {
+	t.Helper()
+
+	if Truef(t, value, msg, args...) {
+		return
+	}
+
+	t.FailNow()
+}
+
+func False(t *testing.T, value bool, msgAndArgs ...interface{}) bool {
+	if value {
+		t.Helper()
+		return fail(t, "Should be false", msgAndArgs...)
+	}
+
+	return true
+}
+
+func RequireFalse(t *testing.T, value bool, msgAndArgs ...interface{}) {
+	t.Helper()
+
+	if False(t, value, msgAndArgs...) {
+		return
+	}
+
+	t.FailNow()
+}
+
+func Empty(t *testing.T, object interface{}, msgAndArgs ...interface{}) bool {
+	pass := isEmpty(object)
+	if !pass {
+		t.Helper()
+		return fail(t, fmt.Sprintf("Should be empty, but was %v", object), msgAndArgs...)
+	}
+
+	return pass
+}
+
+func NotEmpty(t *testing.T, object interface{}, msgAndArgs ...interface{}) bool {
+	pass := !isEmpty(object)
+	if !pass {
+		t.Helper()
+		return fail(t, fmt.Sprintf("Should NOT be empty, but was %v", object), msgAndArgs...)
+	}
+
+	return pass
+}
```

Comment:
I suggest to shorten this too.

Reply:
Noted. I will make the necessary changes.